toolbox/_common.py

```diff
@@ -59,11 +59,12 @@ def run_ansible_playbook(playbook_name, opts: dict = dict()):
     artifact_dir = Path(os.environ["ARTIFACT_DIR"])
     artifact_dir.mkdir(parents=True, exist_ok=True)
 
-    previous_extra_count = len(list(artifact_dir.glob("*__*")))
-    os.environ["ARTIFACT_EXTRA_LOGS_DIR"] = str(
+    if os.environ.get("ARTIFACT_EXTRA_LOGS_DIR") is None:
```

Comment:
The original `_common.sh` file is still in use by some scripts so you might want to update it as well

Reply:
ack, fixed